src/test/java/POM/AllSongsPage.java

```diff
@@ -0,0 +1,36 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+
+public class AllSongsPage extends BasePage {
+    @FindBy(xpath ="//input[@type='search']")
+    WebElement searchField;
+    @FindBy(xpath = "//ul/article/span[2]/span[1]")
+    WebElement mySelectedSong;
+    @FindBy(xpath = "//span[text()='Play']")
+    WebElement playButton;
+    @FindBy(xpath = "//img[@alt='Sound bars']")
```

Comment:
**I recommend:**  make all the locators private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation.